app/templates/components/stream-feed/create-post.hbs

```diff
@@ -102,6 +102,15 @@
         <span class="custom-control-indicator"></span>
         <span class="custom-control-description">Spoiler</span>
       </label>
+
+      {{#if _usableMedia}}
+        {{one-way-select spoiledUnit
```

Comment:
Not sure what to use for UI here. A search field might be too much. Maybe just a number input like in the library?